packages/toolkit/src/mapBuilders.ts

```diff
@@ -166,7 +166,7 @@ export function executeReducerBuilderCallback<S>(
       }
       if (type in actionsMap) {
         throw new Error(
-          '`builder.addCase` cannot be called with two reducers for the same action type'
+          '`builder.addCase` cannot be called with two reducers for the same action type ' + type
```

Comment:
:hand: Can you make a couple tweaks here?

- Change the error string to be a template with backticks
- Use that to put quotes around the `type` variable in the string

(Since this error string _already_ has backticks in it, you'll have to escape those too.)

That way it's more clear what the actual string type value is.